### PR TITLE
Add more docs

### PR DESCRIPTION
**Pull request by samuelallan72**
- add info for working with the snap locally to the readme
- improve user documentation (in snapcraft.yaml)
- add security docs
- move things around for a better split between dev docs (readme) and user docs (snap listing description in snapcraft.yaml)

### CODE REVIEW

File: snap/snapcraft.yaml
Comment (samuelallan72) [inline snap/snapcraft.yaml:15]: Note: we can't use headings, `inline code`, or `[](urls)` in snap store markdown, hence some slightly odd markup here.

Comment (samuelallan72) [reply snap/snapcraft.yaml:15]: Actually I think we can use inline code.

File: README.md
Comment (chanchiwai-ray) [inline README.md:10]: is this correct command? or this one `snapcraft pack`

File: snap/snapcraft.yaml
Comment (chanchiwai-ray) [inline snap/snapcraft.yaml:19]: I think these information are also useful in `README.md` (at least mentioning not all features are available in the snap). To avoid duplication, I suggest add a link in the `README.md` pointing to snapcraft listing (or some thing similar)

**Review by chanchiwai-ray — COMMENTED** [review]
Some questions

File: README.md
Comment (samuelallan72) [reply README.md:10]: Yep this is correct. :)

File: snap/snapcraft.yaml
Comment (samuelallan72) [reply snap/snapcraft.yaml:19]: Good point thanks, I added a note about the user vs developer docs and a link to the snap store listing. Note that the snap store page is a 404 right now - I removed the old releases to tidy things up. It will be there again once we get approval to publish the new snap. :)

Comment (jneo8) [reply snap/snapcraft.yaml:19]: Maybe a known issues doc/page? Since there are some issue required to permission. A link on README can be useful.

**Review by jneo8 — APPROVED** [review]
One non-block suggestion.

File: snap/snapcraft.yaml
Comment (samuelallan72) [reply snap/snapcraft.yaml:19]: That's a good idea - I can expand on them and add more information later (maybe during testing this pulse).

Comment (chanchiwai-ray) [reply snap/snapcraft.yaml:21]: Thanks

File: README.md
Comment (chanchiwai-ray) [reply README.md:13]: yep, note `charmcraft` does not equal to `charmcraft pack`

**Review by chanchiwai-ray — APPROVED** [review]

File: snap/snapcraft.yaml
Comment (Deezzir) [inline snap/snapcraft.yaml:7]: The source-code label is missing as well

Comment (samuelallan72) [reply snap/snapcraft.yaml:7]: Thanks! Added. :)

**Review by jneo8 — APPROVED** [review]

**Review by chanchiwai-ray — APPROVED** [review]

File: snap/snapcraft.yaml
Comment (samuelallan72) [reply snap/snapcraft.yaml:14]: Thanks, yep we can iterate on the docs. For now I was focused on simply getting all the relevent information written down. :)

**Pull request merged**
